### ISSUE TITLE
Handle HTTP 422 response in the feedback component

### ISSUE THREAD

**Issue by kr8n3r** [alphagov/govuk_publishing_components#915]
The feedback component in alphagov/feedback app receives a 422 response ([Unprocessable Entity](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422)) if the user leaves all fields empty. Code [here](https://github.com/alphagov/feedback/blob/20f46dd1f69f389f368484c77e33703a91ce9456/app/controllers/contact/govuk/problem_reports_controller.rb) 

While the error message is displayed, it's  not particularly useful and is at odds with [WCAG SC: 3.3.1](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#minimize-error-identified)

The feedback app response on submissions is 
```
errors: ["What wrong can't be blank", "What doing can't be blank"]
0: "What wrong can't be blank"
1: "What doing can't be blank"
message: "<h2>Sorry, we're unable to send your message as you haven't given us any information.</h2> <p>Please tell us what you were doing or what went wrong.</p>"
status: "error"
```
so ideally the component would cater for that use case in [here](https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/javascripts/govuk_publishing_components/components/feedback.js#L125)

The service service manual frontend app caters for it 
https://github.com/alphagov/service-manual-frontend/blob/master/app/assets/javascripts/modules/improve-this-page.js#L286
